docs/mono-error.md

```diff
@@ -0,0 +1,119 @@
+= MonoError
```

Comment:
The format for headers needs to be either setext or ATX: http://daringfireball.net/projects/markdown/syntax#header